src/phases/query/QueryStatsVariants.tmpl

```diff
@@ -0,0 +1,157 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: >
+  This is a template of predefined variants for QueryStats workloads.
+
+GlobalDefaults:
+  # Local constants.
+  PhasesPerStage: &PhasesPerStage 6
+  SetupPhases: &SetupPhases 2
+
+  # Template variables.
+  Name: &Name {^Parameter: {Name: Name, Default: {unused: "Need to specify the stage name."}}}
+  EntriesPerObject: &EntriesPerObject {^Parameter: {Name: EntriesPerObject, Default: {^RandomInt: {min: 1, max: 350}}}}
+  Group: &Group {^Parameter: {Name: Group, Default: 1}}
+  StageInGroup: &StageInGroup {^Parameter: {Name: StageInGroup, Default: 1}}
+  # For if there needs to be additional initial phases.
+  InitialPhase: &InitialPhase {^Parameter: {Name: InitialPhase, Default: *SetupPhases}}
+  NumGroups: &NumGroups {^Parameter: {Name: NumGroups, Default: {unused: "Need to specify the number of groups being run."}}}
+  StagesPerGroup: &StagesPerGroup {^Parameter: {Name: StagesPerGroup, Default: {unused: "Need to specify the number of stages per group."}}}
+
+  # Syntactic sugar.
+  CycleLength: &CycleLength {^Parameter: {Name: CycleLength, Default: {unused: "Need to provide the cycle length."}}}
```

Comment:
I think I understand what the cycle length is based on our slack history, but for future readers, what is a "cycle"?